show/main.py

```diff
@@ -1228,11 +1228,16 @@ def users(verbose):
 @cli.command()
 @click.option('--since', required=False, help="Collect logs and core files since given date")
 @click.option('--verbose', is_flag=True, help="Enable verbose output")
-def techsupport(since, verbose):
+@click.option('--allow-process-stop', is_flag=True, help="Allow hardware register dump even with system interruption")
```

Comment:
Suggest changing help message to:

```
Also perform hardware register dump which may cause system interruption
```

Reply:
Can you change the message to what I provided above? You changed some, but not all.

Reply:
@sujinmkang: Can you please update this message to the message I provided above? (Note that I also just edited the message again).

Reply:
hardware registers dump can be done even without this option. I will just update the message with "Dump additional data which may require system interruption"